ueberzug:  error: OS unsupported while attempting a build

### Nixpkgs version

- Unstable (25.11)

### Describe the bug

Attempting to perform a system build/switch and there is a failure from ueberzug while building:

ueberzug> [1/5] Compiling C object X.cpython-313-darwin.so.p/ueberzug_X_X.c.o.oK
ueberzug> FAILED: [code=1] X.cpython-313-darwin.so.p/ueberzug_X_X.c.o
ueberzug> clang -IX.cpython-313-darwin.so.p -I. -I.. -I/nix/store/0689dw63w8r55pbxw6d5wan6ry4bggkn-libx11-1.8.12-dev/include -I/nix/store/rji7p7ysb54zmg0b8z9dwybcxyn1v7pg-xorgproto-2024.1/include -I/nix/store/bg0mn0q4a2qam8855kp5cz9vf8373yxz-libxext-1.3.6-dev/include -I/nix/store/yhbbgi03g12syk63q1h3w7aw94s2q0kx-libXres-1.2.2-dev/include -I/nix/store/9px7zzsfnlmj61yy2fpl6z14w93j05is-python3-3.13.6/include/python3.13 -fvisibility=hidden -fdiagnostics-color=always -DNDEBUG -Wall -Winvalid-pch -O3 -MD -MQ X.cpython-313-darwin.so.p/ueberzug_X_X.c.o -MF X.cpython-313-darwin.so.p/ueberzug_X_X.c.o.d -o X.cpython-313-darwin.so.p/ueberzug_X_X.c.o -c ../ueberzug/X/X.c
ueberzug> In file included from ../ueberzug/X/X.c:1:
ueberzug> ../ueberzug/X/python.h:5:2: error: OS unsupported
ueberzug>     5 | #error OS unsupported
ueberzug>       |  ^
ueberzug> 1 error generated.

This is the start of a number of related, cascading failures

### Steps to reproduce

Update to the latest packages and attempt a rebuild. This is using the latest packages pulled this morning.


### Expected behaviour

Build should succeed.

### Screenshots

_No response_

### Relevant log output

```console
ueberzug> [1/5] Compiling C object X.cpython-313-darwin.so.p/ueberzug_X_X.c.o.oK
ueberzug> FAILED: [code=1] X.cpython-313-darwin.so.p/ueberzug_X_X.c.o
ueberzug> clang -IX.cpython-313-darwin.so.p -I. -I.. -I/nix/store/0689dw63w8r55pbxw6d5wan6ry4bggkn-libx11-1.8.12-dev/include -I/nix/store/rji7p7ysb54zmg0b8z9dwybcxyn1v7pg-xorgproto-2024.1/include -I/nix/store/bg0mn0q4a2qam8855kp5cz9vf8373yxz-libxext-1.3.6-dev/include -I/nix/store/yhbbgi03g12syk63q1h3w7aw94s2q0kx-libXres-1.2.2-dev/include -I/nix/store/9px7zzsfnlmj61yy2fpl6z14w93j05is-python3-3.13.6/include/python3.13 -fvisibility=hidden -fdiagnostics-color=always -DNDEBUG -Wall -Winvalid-pch -O3 -MD -MQ X.cpython-313-darwin.so.p/ueberzug_X_X.c.o -MF X.cpython-313-darwin.so.p/ueberzug_X_X.c.o.d -o X.cpython-313-darwin.so.p/ueberzug_X_X.c.o -c ../ueberzug/X/X.c
ueberzug> In file included from ../ueberzug/X/X.c:1:
ueberzug> ../ueberzug/X/python.h:5:2: error: OS unsupported
ueberzug>     5 | #error OS unsupported
ueberzug>       |  ^
ueberzug> 1 error generated.
ueberzug> [2/5] Compiling C object X.cpython-313-darwin.so.p/ueberzug_X_display.c.o
ueberzug> FAILED: [code=1] X.cpython-313-darwin.so.p/ueberzug_X_display.c.o
ueberzug> clang -IX.cpython-313-darwin.so.p -I. -I.. -I/nix/store/0689dw63w8r55pbxw6d5wan6ry4bggkn-libx11-1.8.12-dev/include -I/nix/store/rji7p7ysb54zmg0b8z9dwybcxyn1v7pg-xorgproto-2024.1/include -I/nix/store/bg0mn0q4a2qam8855kp5cz9vf8373yxz-libxext-1.3.6-dev/include -I/nix/store/yhbbgi03g12syk63q1h3w7aw94s2q0kx-libXres-1.2.2-dev/include -I/nix/store/9px7zzsfnlmj61yy2fpl6z14w93j05is-python3-3.13.6/include/python3.13 -fvisibility=hidden -fdiagnostics-color=always -DNDEBUG -Wall -Winvalid-pch -O3 -MD -MQ X.cpython-313-darwin.so.p/ueberzug_X_display.c.o -MF X.cpython-313-darwin.so.p/ueberzug_X_display.c.o.d -o X.cpython-313-darwin.so.p/ueberzug_X_display.c.o -c ../ueberzug/X/display.c
ueberzug> In file included from ../ueberzug/X/display.c:1:
ueberzug> ../ueberzug/X/python.h:5:2: error: OS unsupported
ueberzug>     5 | #error OS unsupported
ueberzug>       |  ^
ueberzug> 1 error generated.
ueberzug> [3/5] Compiling C object X.cpython-313-darwin.so.p/ueberzug_X_Xshm.c.o
ueberzug> FAILED: [code=1] X.cpython-313-darwin.so.p/ueberzug_X_Xshm.c.o
ueberzug> clang -IX.cpython-313-darwin.so.p -I. -I.. -I/nix/store/0689dw63w8r55pbxw6d5wan6ry4bggkn-libx11-1.8.12-dev/include -I/nix/store/rji7p7ysb54zmg0b8z9dwybcxyn1v7pg-xorgproto-2024.1/include -I/nix/store/bg0mn0q4a2qam8855kp5cz9vf8373yxz-libxext-1.3.6-dev/include -I/nix/store/yhbbgi03g12syk63q1h3w7aw94s2q0kx-libXres-1.2.2-dev/include -I/nix/store/9px7zzsfnlmj61yy2fpl6z14w93j05is-python3-3.13.6/include/python3.13 -fvisibility=hidden -fdiagnostics-color=always -DNDEBUG -Wall -Winvalid-pch -O3 -MD -MQ X.cpython-313-darwin.so.p/ueberzug_X_Xshm.c.o -MF X.cpython-313-darwin.so.p/ueberzug_X_Xshm.c.o.d -o X.cpython-313-darwin.so.p/ueberzug_X_Xshm.c.o -c ../ueberzug/X/Xshm.c
ueberzug> In file included from ../ueberzug/X/Xshm.c:1:
ueberzug> ../ueberzug/X/python.h:5:2: error: OS unsupported
ueberzug>     5 | #error OS unsupported
ueberzug>       |  ^
ueberzug> ../ueberzug/X/Xshm.c:194:34: error: call to undeclared function 'max'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
ueberzug>   194 |     destination_offset_x_bytes = max(0, offset_x) * BYTES_PER_PIXEL;
ueberzug>       |                                  ^
ueberzug> ../ueberzug/X/Xshm.c:194:34: note: did you mean 'fmax'?
ueberzug> /nix/store/7jphm2f6rg034wdlm76mn4hifz3knzrw-apple-sdk-11.3/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/math.h:532:15: note: 'fmax' declared here
ueberzug>   532 | extern double fmax(double, double);
ueberzug>       |               ^
ueberzug> ../ueberzug/X/Xshm.c:197:22: error: call to undeclared function 'min'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
ueberzug>   197 |     pixels_per_row = min(width + min(offset_x, 0), self->width - max(offset_x, 0)) * BYTES_PER_PIXEL;
ueberzug>       |                      ^
ueberzug> 3 errors generated.
ueberzug> [4/5] Compiling C object X.cpython-313-darwin.so.p/ueberzug_X_window.c.o
ueberzug> FAILED: [code=1] X.cpython-313-darwin.so.p/ueberzug_X_window.c.o
ueberzug> clang -IX.cpython-313-darwin.so.p -I. -I.. -I/nix/store/0689dw63w8r55pbxw6d5wan6ry4bggkn-libx11-1.8.12-dev/include -I/nix/store/rji7p7ysb54zmg0b8z9dwybcxyn1v7pg-xorgproto-2024.1/include -I/nix/store/bg0mn0q4a2qam8855kp5cz9vf8373yxz-libxext-1.3.6-dev/include -I/nix/store/yhbbgi03g12syk63q1h3w7aw94s2q0kx-libXres-1.2.2-dev/include -I/nix/store/9px7zzsfnlmj61yy2fpl6z14w93j05is-python3-3.13.6/include/python3.13 -fvisibility=hidden -fdiagnostics-color=always -DNDEBUG -Wall -Winvalid-pch -O3 -MD -MQ X.cpython-313-darwin.so.p/ueberzug_X_window.c.o -MF X.cpython-313-darwin.so.p/ueberzug_X_window.c.o.d -o X.cpython-313-darwin.so.p/ueberzug_X_window.c.o -c ../ueberzug/X/window.c
ueberzug> In file included from ../ueberzug/X/window.c:1:
ueberzug> ../ueberzug/X/python.h:5:2: error: OS unsupported
ueberzug>     5 | #error OS unsupported
ueberzug>       |  ^
ueberzug> 1 error generated.
ueberzug> ninja: build stopped: subcommand failed.
ueberzug>
ueberzug> ERROR Backend subprocess exited when trying to invoke build_wheel
error: Cannot build '/nix/store/kkz23ayar8d22lbshx85azjp7j17ncw0-python3.13-ueberzug-18.3.1.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/ry3fwfnxanxlrsfnmq6jy7z9dn7wsga0-python3.13-ueberzug-18.3.1-dist
         /nix/store/z6vppbnzh435vd1l75nnvpwv0ggimnzp-python3.13-ueberzug-18.3.1
       Last 25 log lines:
       > ../ueberzug/X/python.h:5:2: error: OS unsupported
       >     5 | #error OS unsupported
       >       |  ^
       > ../ueberzug/X/Xshm.c:194:34: error: call to undeclared function 'max'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
       >   194 |     destination_offset_x_bytes = max(0, offset_x) * BYTES_PER_PIXEL;
       >       |                                  ^
       > ../ueberzug/X/Xshm.c:194:34: note: did you mean 'fmax'?
       > /nix/store/7jphm2f6rg034wdlm76mn4hifz3knzrw-apple-sdk-11.3/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/math.h:532:15: note: 'fmax' declared here
       >   532 | extern double fmax(double, double);
       >       |               ^
       > ../ueberzug/X/Xshm.c:197:22: error: call to undeclared function 'min'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
       >   197 |     pixels_per_row = min(width + min(offset_x, 0), self->width - max(offset_x, 0)) * BYTES_PER_PIXEL;
       >       |                      ^
       > 3 errors generated.
       > [4/5] Compiling C object X.cpython-313-darwin.so.p/ueberzug_X_window.c.o
       > FAILED: [code=1] X.cpython-313-darwin.so.p/ueberzug_X_window.c.o
       > clang -IX.cpython-313-darwin.so.p -I. -I.. -I/nix/store/0689dw63w8r55pbxw6d5wan6ry4bggkn-libx11-1.8.12-dev/include -I/nix/store/rji7p7ysb54zmg0b8z9dwybcxyn1v7pg-xorgproto-2024.1/include -I/nix/store/bg0mn0q4a2qam8855kp5cz9vf8373yxz-libxext-1.3.6-dev/include -I/nix/store/yhbbgi03g12syk63q1h3w7aw94s2q0kx-libXres-1.2.2-dev/include -I/nix/store/9px7zzsfnlmj61yy2fpl6z14w93j05is-python3-3.13.6/include/python3.13 -fvisibility=hidden -fdiagnostics-color=always -DNDEBUG -Wall -Winvalid-pch -O3 -MD -MQ X.cpython-313-darwin.so.p/ueberzug_X_window.c.o -MF X.cpython-313-darwin.so.p/ueberzug_X_window.c.o.d -o X.cpython-313-darwin.so.p/ueberzug_X_window.c.o -c ../ueberzug/X/window.c
       > In file included from ../ueberzug/X/window.c:1:
       > ../ueberzug/X/python.h:5:2: error: OS unsupported
       >     5 | #error OS unsupported
       >       |  ^
       > 1 error generated.
       > ninja: build stopped: subcommand failed.
       >
       > ERROR Backend subprocess exited when trying to invoke build_wheel
       For full logs, run:
         nix log /nix/store/kkz23ayar8d22lbshx85azjp7j17ncw0-python3.13-ueberzug-18.3.1.drv
error: Cannot build '/nix/store/r2s5hx4805llr1cyjqyndxyw3bhmzz99-neofetch-unstable-2021-12-10.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/1wvhn3b7rr7hdarkpx7avfb65ijafpfk-neofetch-unstable-2021-12-10-man
         /nix/store/l7jszkwgww0ax2zwy2qdbvc90bggw7kj-neofetch-unstable-2021-12-10
error: Cannot build '/nix/store/l06v5vik1qqyvdgwxfk23ymdb4ih0qk1-system-applications.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/rz76rz5l0ycjr9g0c5mi5q9gw60w2jaq-system-applications
error: Cannot build '/nix/store/lviqnfbpz1pypvflwbshr8jnyci9qb09-system-path.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/jz1n9pxpqf34scg5x2qj9rw5qh5lyqgj-system-path
error: Cannot build '/nix/store/5hgm9sd0bqa9akwr980xrvm2w0kmp51k-darwin-system-25.11.8df64f8.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/wvp197893n40wi3mad8wlhkixffnq1vr-darwin-system-25.11.8df64f8
┏━ 5 Errors:
 ⋮
┃        > ../ueberzug/X/Xshm.c:197:22: error: call to undeclared function 'min'; ISO C99 and later do not support implicit function declarations [-Wimplici…
┃        >   197 |     pixels_per_row = min(width + min(offset_x, 0), self->width - max(offset_x, 0)) * BYTES_PER_PIXEL;
┃        >       |                      ^
┃        > 3 errors generated.
┃        > [4/5] Compiling C object X.cpython-313-darwin.so.p/ueberzug_X_window.c.o
┃        > FAILED: [code=1] X.cpython-313-darwin.so.p/ueberzug_X_window.c.o
┃        > clang -IX.cpython-313-darwin.so.p -I. -I.. -I/nix/store/0689dw63w8r55pbxw6d5wan6ry4bggkn-libx11-1.8.12-dev/include -I/nix/store/rji7p7ysb54zmg0b8…
┃        > In file included from ../ueberzug/X/window.c:1:
┃        > ../ueberzug/X/python.h:5:2: error: OS unsupported
┃        >     5 | #error OS unsupported
┃        >       |  ^
┃        > 1 error generated.
┃        > ninja: build stopped: subcommand failed.
┃        >
┃        > ERROR Backend subprocess exited when trying to invoke build_wheel
┃        For full logs, run:
┃          nix log /nix/store/kkz23ayar8d22lbshx85azjp7j17ncw0-python3.13-ueberzug-18.3.1.drv
```

### Additional context

_No response_

### System metadata

❯ nix-shell -p nix-info --run "nix-info -m"
unpacking 'https://flakehub.com/f/DeterminateSystems/nixpkgs-weekly/%2A.tar.gz' into the Git cache...
these 2 paths will be fetched (0.01 MiB download, 0.10 MiB unpacked):
  /nix/store/jaq91nmv6g5smvspb579b825syavpczy-DarwinTools-1
  /nix/store/07cmv1nfzq1pwm4wkaa0l1fvv7k0r1aw-nix-info
copying path '/nix/store/jaq91nmv6g5smvspb579b825syavpczy-DarwinTools-1' from 'https://cache.nixos.org'...
copying path '/nix/store/07cmv1nfzq1pwm4wkaa0l1fvv7k0r1aw-nix-info' from 'https://cache.nixos.org'...
 - system: `"aarch64-darwin"`
 - host os: `Darwin 24.6.0, macOS 15.6.1`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.30.2`
 - channels(root): `""`
 - nixpkgs: `/nix/store/amab8crkp25ncy8q6qxvi9jrws3ak63d-source`


### Are you using nix-darwin?

Yes, I am using nix-darwin.

### Notify maintainers

@Br1ght0ne 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+bug%22+label%3A%226.topic%3A+darwin%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ne/neofetch/package.nix
+++ b/pkgs/by-name/ne/neofetch/package.nix
@@ -5,7 +5,7 @@
   bash,
   makeWrapper,
   pciutils,
-  x11Support ? !stdenvNoCC.hostPlatform.isOpenBSD,
+  x11Support ? stdenvNoCC.hostPlatform.isLinux,
   ueberzug,
   fetchpatch,
 }:

